### DISCUSSION TITLE
App access to all users but not public

### DISCUSSION THREAD

**Discussion by mielune** [Budibase/budibase#7245] [Ideas]
Hi folks,

I'm build an app for all my company people and I would like to allow app access to everybody, but authenticated for security and audit needs.
I don't find a setting to do that.

To handle this feature, I see 2 cases:
- Add an access type "Authenticate" in the roles to allow all authenticated users to use this data/screens
- Add a special entry in the Access Management of the apps to setup a role (Basic in my case) to all authenticated users automatically

Open to discuss about this with you :)

**Comment by melohagan** [comment]
Hey @mielune 

I think this may be a related issue: https://github.com/Budibase/budibase/issues/7269

When user groups are made available in the next release, this may provide what you need!
